projects/packages/sync/src/class-replicastore.php

```diff
@@ -1086,10 +1086,11 @@ public function user_count() {
 	 * @access public
 	 *
 	 * @param int $user_id User ID.
-	 * @return \WP_User User object.
+	 * @return \WP_User|null User object, or `null` if user invalid/not found.
```

Comment:
Not a blocker, but probably should update the doc in the interface as well?

Reply:
Good suggestion. 2d26deef3c20ea71a1e882761fe7833f44f2e4b0